src/containers/item-details/index.tsx

```diff
@@ -17,12 +21,33 @@ type Props = {
 };
 
 const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
+  const { query } = useRouter();
+  const { itemId } = query;
+  const { ethAddress } = useAppSelector(selectAuthData);
+
   const [selectedImg, setSelectedImg] = useState(item.image.url);
 
+  const [itemOwnerAddress, setItemOwnerAddress] = useState('');
+  const [itemPinataUrl, setItemPinataUrl] = useState('');
+
   const handleImageChange = (img: string) => {
     setSelectedImg(img);
   };
 
+  /* const handleGetItemById = async (id: string) => {
+    const res = await axios.get(`http://localhost:3333/v1/items/${id}`);
+    setItemOwnerAddress(res.data.owner.address);
+    setItemPinataUrl(res.data.image.url);
+  }; */
+
+  useEffect(() => {
+    (async () => {
+      const res = await axios.get(`http://localhost:3333/v1/items/${itemId}`);
```

Comment:
change _http://localhost:3333/v1/items_ and put _process.env.NEXT_PUBLIC_LEDA_API_URL/_ instead

Reply:
This will prevent that the Deployment falls down when it's up